README.md

```diff
@@ -46,67 +46,122 @@ $ make
 
 ## CLI Usage
 
-For a quick demo, watch: https://www.youtube.com/watch?v=pWRZhZgfYSw 
+```
+$ kbrew --help
+TODO: Long description
+
+Usage:
+  kbrew [command]
+
+Available Commands:
+  analytics   Manage analytics setting
+  completion  Output shell completion code for the specified shell
+  help        Help about any command
+  info        Describe application
+  install     Install application
+  remove      Remove application
+  search      Search application
+  update      Update kbrew and recipe registries
+  version     Print version information
+
+Flags:
+  -c, --config string       config file (default is $HOME/.kbrew.yaml)
+      --config-dir string   config dir (default is $HOME/.kbrew)
+      --debug               enable debug logs
+  -h, --help                help for kbrew
+  -n, --namespace string    namespace
+
+Use "kbrew [command] --help" for more information about a command.
+```
 
-### kbrew install
+For a quick demo, watch: https://www.youtube.com/watch?v=pWRZhZgfYSw 
 
-Installs a recipe in your cluster with all pre & posts steps and applications.
+### Commonly used commands
 
-### kbrew search
+#### kbrew search
 
 Searches for a recipe for the given application. Lists all the available recipes if no application name is passed.
 
-### kbrew update
+#### kbrew info
+
+Prints applications details including registry and dependency information. 
+
+#### kbrew install
+
+Installs a recipe in your cluster with all pre & posts steps and applications.
+
+#### kbrew update
 
 Checks for kbrew updates and upgrades automatically if a newer version is available.
 Fetches updates for all the kbrew recipe registries
 
-### kbrew remove 
+#### kbrew remove 
 
-Partially implemented as of now.
+Uninstalls the application and its dependencies.
 
-## Terminology
+## Workflow
```

Comment:
This should be called `Working of a Kbrew Recipe` or `Genesis of a Kbrew Recipe` 
Workflow is very overloaded and does not convey clearly what we want to say IMHO.